src/main/java/app/controller/io/FileParser.java

```diff
@@ -29,6 +30,10 @@ public static void parseNextLine(Settings settings, String nextLine, int countLi
             {
                 String id=scan.next();
                 String val=scan.next();
+                if(val.contains("//"))
```

Comment:
Perfectly fine implementation, but why not abstract this to a separate method?
```java
line = removeComments(line);
```
Improves readability and reusability of code

Reply:
will do